src/styles/sidebar.styl

```diff
@@ -24,7 +24,7 @@
 
     // need this due to no css-modules usage in this app
     aside
-        width   100%!important
+        width   100%!important // @stylint ignore
```

Comment:
In fact this one doesn't give me the choice :/. I have two other ignore comment but it's just until we use css-modules in this app.